tests/integ/modin/index/test_index_methods.py

```diff
@@ -11,6 +11,8 @@
 from tests.integ.modin.sql_counter import SqlCounter, sql_count_checker
 from tests.integ.modin.utils import assert_index_equal, assert_series_equal
 
+data = {"col1": [1, 2, 3], "col2": [3, 4, 5], "col3": [5, 6, 7]}
+
 NATIVE_INDEX_TEST_DATA = [
```

Comment:
Can we add tests with an Index object has has Nan/Null values as well? Nulls are including in size/shape/empty so the code shouldn't change but it's still good to have the test for future APIs as well.

Reply:
does your index object also support having list values? Something like pd.Index([[1, 2], [2, 3], [3, 4]]). If yes, add tests for that as well.